05-html-css/flexbox-intro.md

```diff
@@ -0,0 +1,230 @@
+# Intro to Alignment and Positioning with Flexbox
+
+## Learning Goals
+- Understand the historical context of flexbox compared to other methods of aligning and positioning with CSS
+- Review how the browser uses _flow_ to place elements on the page
+- Change how elements flow on the page using CSS's `display: flex` property
+- Change how elements align on the page using CSS's `display: flex` property
+- Know browser compatibility for flexbox
+
+## Before We Begin: Review Display
+
+When rendering, the browser _flows_ from left to right, placing things in a single line. The line is as tall as the tallest element, and all the things are lined up with its bottom. If a thing would go off the edge of the screen, the browser wraps around and starts a new line, completely below the current one.
```

Comment:
This section is a copy-paste of the old display lesson, right? I think the visuals are valuable enough that we should keep that as a separate resource, and link to it at the top of this lesson. We don't need to dive deep, a few minutes should be plenty especially since they're reading it as homework (I think?), but pictures always help!

Reply:
decided to keep it in because the instructor should just zoom through it if it's just review annnnd i'm not gonna get around to wordsmithing this right now